### PR TITLE
feat(tutorial): complete step 1 Hursley

### PR DESCRIPTION
**Pull request by Tobias-Hunter**

### CODE REVIEW

File: src/components/TutorialHeader/TutorialHeader.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  HeaderGlobalAction,
+  SkipToContent,
+} from 'carbon-components-react/lib/components/UIShell';
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
```
Comment (kellychurchill) [inline src/components/TutorialHeader/TutorialHeader.js]: You're missing the `<HeaderNavigation>` component. See https://www.carbondesignsystem.com/tutorial/react-step-1#import-and-export-the-header

Comment (Tobias-Hunter) [reply src/components/TutorialHeader/TutorialHeader.js]: All done, don't know how I missed that line out 😢

Comment (kellychurchill) [reply src/components/TutorialHeader/TutorialHeader.js]: Whoops - you also missed the routing updates: https://www.carbondesignsystem.com/tutorial/react-step-1/#add-routing

Comment (Tobias-Hunter) [reply src/components/TutorialHeader/TutorialHeader.js]: 3rd times a charm 😂

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**